spec/controllers/membership_controller_spec.rb

```diff
@@ -113,5 +113,19 @@ def do_patch
       expect(assigns(:membership).application_memberships.size).to eq(1)
       expect(assigns(:membership).application_memberships.first.application_id).to eq(application1.id)
     end
+
+    context "when the can_only_grant_own_roles feature is enabled" do
+      before do
+        allow(FeatureFlags::Features).to receive(:enabled?).with("can_only_grant_own_roles").and_return(false)
```

Comment:
the test description in the ```context``` says when the feature is *enabled* but the stubbing returns ```false```.